files/etc/init/mistral.uwsgi.conf

```diff
@@ -0,0 +1,7 @@
+description "OpenStack Workflow Service"
+start on runlevel [2345]
+stop on runlevel [016]
+respawn
+script
+    /opt/openstack/mistral/.venv/bin/python /opt/openstack/mistral/mistral/cmd/launch.py --server engine,executor --config-file /etc/mistral/mistral.conf --log-file /var/log/mistral_api.log &> /dev/null &
```

Comment:
Shouldn't this be a uwsgi call pointing to the wsgi file in /opt/openstack/mistral?  This looks like the standard mistral launch command using simple server.